internal/projects/create.go

```diff
@@ -70,7 +76,11 @@ func ProvisionSelfEnrolledProject(
 		Metadata: jsonmeta,
 	})
 	if err != nil {
-		return nil, status.Errorf(codes.Internal, "failed to create default project: %v", err)
+		// Check if `project_name_lower_idx` unique constraint was violated
```

Comment:
Can you use `ErrIsUniqueViolation` instead?

Reply:
Do you want me to rename the error to that? or do you want me to use `ErrIsUniqueViolation` to detect the error? I don't find that error being provided by the SQL package.

Reply:
Sorry I was unclear, I wanted to ask if you think that using https://github.com/stacklok/minder/blob/main/internal/db/errors.go#L24 would be cleaner than checking for the string in Error

Reply:
Oooh! I forgot that existed